stdlib/public/Platform/POSIXError.swift

```diff
@@ -556,6 +556,168 @@ public enum POSIXErrorCode : Int32 {
   case EHWPOISON       = 133
 }
 
+#elseif os(WASI)
+
+/// Enumeration describing POSIX error codes.
+public enum POSIXErrorCode : Int32 {
```

Comment:
Since WASI is not POSIX, shouldn't this be named `WASIErrorCode`?

Reply:
WASI doesn't provide POSIX API, but wasi-libc is clearly POSIX compatible according to their readme.
> WASI Libc is sufficiently stable and usable for many purposes, as most of the POSIX-compatible APIs are stable, though it is continuing to evolve to better align with wasm and WASI.

https://github.com/WebAssembly/wasi-libc

Reply:
Cool, thanks for the clarification.